src/features/unified-share-modal/SharedLinkPermissionMenu.js

```diff
@@ -136,4 +143,8 @@ class SharedLinkPermissionMenu extends Component<Props> {
     }
 }
 
-export default SharedLinkPermissionMenu;
+const sharedLinkPermissionMenu = withMediaQuery(SharedLinkPermissionMenu);
+sharedLinkPermissionMenu.displayName = 'SharedLinkPermissionMenu';
+
+export { SharedLinkPermissionMenu as SharedLinkPermissionMenuBase };
+export default sharedLinkPermissionMenu;
```

Comment:
The above with 

```
export { SharedLinkPermissionMenu as SharedLinkPermissionMenuBase };
export default withMediaQuery(SharedLinkPermissionMenu);
```

was how I had it originally. However, when the snapshots render it gives it as generic `Component` which I felt was not very useful so I updated the display name.

Changing it in `withMediaQuery` is a possibility, but I think it should be a separate PR since we may have to update other snapshots as well.